dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/common/ExtractionPipelineBuilder.scala

```diff
@@ -77,6 +77,7 @@ class ExtractionPipelineBuilder(
           arm = arms
         )
       )
+
```

Comment:
_Very_ small nitpick -- I generally avoid unwanted whitespace changes like this to keep the history for this file clean.